stock_picking_invoice_link/models/stock_move.py

```diff
@@ -3,15 +3,27 @@
 # Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
-from odoo import fields, models
+from odoo import _, fields, models
+from odoo.exceptions import UserError
 
 
 class StockMove(models.Model):
     _inherit = "stock.move"
 
-    invoice_line_id = fields.Many2one(
+    invoice_line_ids = fields.Many2many(
         comodel_name='account.invoice.line',
+        relation='stock_move_invoice_line_rel',
+        column1='move_id',
+        column2='invoice_line_id',
         string='Invoice Line',
         copy=False,
         readonly=True,
     )
+
+    def write(self, vals):
```

Comment:
Please a docstring explaining why you are preventing this.

Reply:
Reading the current docstring, shouldn't it block `qty_done` instead of `product_uom_qty`?

Reply:
for a picking in done state when user modifies the units... the field product_uom_qty is writed too.